ewcmd.py

```diff
@@ -845,7 +899,7 @@ def weather_txt(id_server):
 			
 			ewitem.item_delete(id_item = item.id_item)
 
-			pay = int(random.random() * 10 ** random.randrange(2,5))
+			pay = int(random.random() * 100 ** random.randrange(2,5))
```

Comment:
operator priority! this increases maximum output by a factor of 10^4. increase to randrange(2,6) instead

Reply:
Understood. Dodged a bullet there lol...